evalml/data_checks/highly_null_data_check.py

```diff
@@ -45,7 +45,7 @@ def validate(self, X, y=None):
                                                                  "data_check_name": "HighlyNullDataCheck",\
                                                                  "level": "warning",\
                                                                  "code": "HIGHLY_NULL",\
-                                                                 "details": {"column": "lots_of_null"}}],\
+                                                                 "details": {"column": "lots_of_null", "pct_null_rows": 0.8}}],\
```

Comment:
Open to suggestion: keep as 0.8 or display as percentage 80%?

Reply:
Maybe display as percentage since the field name has the word percent in it? I don't feel super strongly about it though

Reply:
I think 0.8 is better for now. I believe use [0, 1] notation for percent baseline, class imbalance ratios in params and other fields. I think its nice having our codebase be consistent with how we represent percentages.